src/BenchmarksApps/Kestrel/PlatformBenchmarks/benchmarks.fortunes.yml

```diff
@@ -23,23 +23,65 @@ jobs:
     readyStateText: PostgreSQL init process complete
 
 scenarios:
-  fortunesplatform:
+  fortunes:
```

Comment:
Will changing the name cause issue for the results graphs? (i.e. history)

Reply:
@benaadams  that's a very good question

@sebastienros can I change this name? I just want to shorten it to make it easier to type the command

Reply:
there is no risk to break anything because we are not recording the results of this file, but the old json ones. However it will once we start using the yml files. And we don't want to include a custom metadata like "platform=true", that wouldn't make sense, I'd rather see it's a platform benchmark in its name.

Reply:
@sebastienros but we know that it's a platform benchmark because the `.yml` file lives in the `PlatformBenchmarks` folder. Imho having "platform" would make more sense if we had one big `.yml` file with all the benchmarks. The same goes for "aspnet" in the profile name. I have access only to aspnet machines, so to me it's redundant

```cmd
--config ..\BenchmarksApps\Kestrel\PlatformBenchmarks\benchmarks.fortunes.yml --profile aspnet-citrine --scenario fortunesplatform
```

```cmd
--config ..\BenchmarksApps\Kestrel\PlatformBenchmarks\benchmarks.fortunes.yml --profile citrine --scenario fortunes
```

It's not super important to me, just a way to make the commands shorter and easier to memorize.